### DISCUSSION TITLE
How do you feel about CodeRabbit for AI code reviews? Vote in this poll!

### DISCUSSION THREAD

**Discussion by Piedone** [OrchardCMS/OrchardCore#15935] [Polls]
As part of https://github.com/OrchardCMS/OrchardCore/issues/15439, we had automatic AI code reviews with [CodeRabbit](https://coderabbit.ai/) for all PRs in the last week. After our experiences, it's time to decide how to move forward. If you've interacted with the CodeRabbit review bot, please vote below, and add comments as well!

Note that you can use [various commands](https://docs.coderabbit.ai/guides/coderabbit-commands) with the bot, as well as there are [config options](https://docs.coderabbit.ai/guides/configure-coderabbit) to tune it to what we need (what I started under https://github.com/OrchardCMS/OrchardCore/pull/15813).

**Comment by Piedone** [comment]
I found that it's most useful for documentation changes (most possibly its training data including a lot of English text and also Markdown). For code changes, only occasionally and mostly it's noise.

I side effect of the bot's comments is that in the [PR list](https://github.com/OrchardCMS/OrchardCore/pulls) it'll show also those PRs as having comments that still need a person's review, making it harder to distinguish. This can be improved with an automatically managed tag though.

**Comment by Skrypt** [comment]
It is going to take a little more time for me to say. So far, I'd say that the 🐰 didn't do much else than doing a summary text of the PR.

**Reply by Piedone** [reply]
Thanks!

**Comment by chagil123** [comment]
@coderabbitai please review this pr
